protocol/triple/internal/client/cmd_instance/main.go

```diff
@@ -36,7 +36,7 @@ func main() {
 	}
 	// configure the params that only client layer cares
 	cli, err := ins.NewClient(
-		client.WithURL("tri://127.0.0.1:20000"),
+		client.WithURL("127.0.0.1:20000"),
```

Comment:
How will the protocol going to be used determined if no protocol is set?

Reply:
So it will check `ref.Protocol`? What if `ref.Protocol` is also not specified.

Reply:
Client would use Triple protocol by default. Actually, the ideal configuration form would be like:
```
cli, err := ins.NewClient(
    client.WithURL("127.0.0.1:20000"),
    client.WithTriple(),
)
```
Since we decided to make Triple as default protocol, there is no need to provide ```client.WithTriple``` to users. Then, there would be other protocols that could be supported by new API in the future, we would provide ```client.WithXXX``` to users.

Reply:
> So it will check `ref.Protocol`? What if `ref.Protocol` is also not specified.
```global.DefaultReferenceConfig``` has set triple as default protocol.
By the way, that is the problem we need to talk. **What is the correct location that we should assign default values?**
For instance, we set default protocol in **global.ReferenceConfig**. But it is too much granularity of impact. I think the right to assign default values should be in the corresponding module.

Reply:
I see. I will create an issue for this to make sure we can get back to this later.